apps/clr-ui-tokens/src/app/combobox-demo/combobox-demo.component.ts

```diff
@@ -0,0 +1,337 @@
+import { Component } from '@angular/core';
+import { timer, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { tap } from 'rxjs/internal/operators';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-combobox-demo',
+  templateUrl: './combobox-demo.component.html',
+  styleUrls: ['./combobox-demo.component.scss'],
```

Comment:
Do we need `apps/clr-ui-tokens/src/app/combobox-demo/combobox-demo.component.scss` if it's an empty file?